scripts/update-bcd.ts

```diff
@@ -679,6 +684,48 @@ const skipCurrentBeforeSupport = skip("currentBeforeSupport", ({
   }
 });
 
+export const hasSupportMatrixContradictions = (
+  versionMap: BrowserSupportMap,
+  simpleStatement?: SimpleSupportStatement,
+) => {
+  if (!simpleStatement || simpleStatement.version_added === null) {
+    return true;
+  }
+
+  if (simpleStatement.version_added === "preview") {
```

Comment:
We can't always update a "preview" version, it's only if we have a true test result. With the change suggested above, I think this would move into a loop, and should be condition on `supportAssertion` being true for some browser version.

Reply:
Addressed in https://github.com/openwebdocs/mdn-bcd-collector/pull/909/commits/1a15d198a36f6f6df4cd8f1f3d928474937e78ef, particularly with [these test cases](https://github.com/openwebdocs/mdn-bcd-collector/pull/909/commits/1a15d198a36f6f6df4cd8f1f3d928474937e78ef#diff-a08001be368203eeaff8a37f3e414fbee6bc35cc0ec545f3e77c01bd22abf06fR954-R966). Let me know if that looks correct!

Reply:
Yes, the handling of "preview" there looks good, and the tests confirms it works!